tests/test_accessibility.py

```diff
@@ -312,6 +317,41 @@ def test_check_for_accessibility_errors(self):
         with self.assertRaises(AccessibilityError):
             self.page.a11y_audit.check_for_accessibility_errors()
 
+    def test_customize_ruleset_via_env_var(self):
+        with patch.dict(os.environ,
+            {'BOKCHOY_A11Y_CUSTOM_RULES_FILE': 'tests/a11y_custom_rules.js'}):
+            self.page.a11y_audit.config.customize_ruleset()
+            self._do_audit_and_check_errors(7)
+
+    def test_customize_ruleset_via_config_param(self):
+        self.page.a11y_audit.config.customize_ruleset('tests/a11y_custom_rules.js')
+        self._do_audit_and_check_errors(7)
+
+    def test_customize_ruleset_bad_file_path(self):
```

Comment:
@benpatterson 
re: "Could you add a test(s) for when the file is not found?"

I just added this test.